### ISSUE TITLE
config.ini

### ISSUE THREAD

**Issue by andyrav** [vanstinator/PlexMonitor#18]
Hi
Can you rename your config.ini file so that it doe not cause merge issue when doing git pull
cheers

error: Your local changes to the following files would be overwritten by merge:
        config.ini
Please, commit your changes or stash them before you can merge.

**Comment by vanstinator** [comment]
No. Just add it to your gitignore. I'll look at doing that in the future
with the main build. I'll also have a bugfix release out later tonight for
your other issue.
